### ISSUE TITLE
Removing rating and numberRates properties

### ISSUE THREAD

**Issue by brett-patterson** [masterT/ultimate-guitar-scraper#11]
https://github.com/masterT/ultimate-guitar-scraper/blob/f55265bb1ed2a1f6a3189d0f7518a0627f413889/lib/utils.js#L105-L109

Is there a reason for this? Right now it always removes the `rating` and `numberRates` properties if one of them is present, resulting in the search results never containing rating information.

**Comment by masterT** [comment]
Ok, that's a mistake, the condition should be :

```js
!tab.hasOwnProperty('rating') || !tab.hasOwnProperty('numberRates')
```

Do you want to do a pull request?

**Comment by masterT** [comment]
Oh, nice! You already fixed it! #12 😄
